aws-sns-topic/src/main/java/software/amazon/sns/topic/UpdateHandler.java

```diff
@@ -90,9 +90,14 @@ protected ProgressEvent<ResourceModel, CallbackContext> handleRequest(
                     return progress;
                 })
                 .then(progress -> {
-                    if(!StringUtils.equals(model.getTracingConfig(), previousModel.getTracingConfig())) {
+                    if ( previousModel.getTracingConfig() == null && model.getTracingConfig() == null) {
+                        return progress;
+                    }
+                    String previousVal = previousModel.getTracingConfig();
+                    String desiredVal =  model.getTracingConfig() != null ? model.getTracingConfig().toString() : "PassThrough";
```

Comment:
Let's create enum for this `TracingConfigStatus`, as Active and `PassThrough`.
The only two status will be more clear, and we can use it in below test